.github/workflows/tests.yaml

```diff
@@ -20,16 +20,13 @@ jobs:
         run: |
           # $CONDA is an environment variable pointing to the root of the miniconda directory
           $CONDA/bin/conda env update --file environment.yml --name base
-          $CONDA/bin/pip install .
+          $CONDA/bin/pip install -e .
```

Comment:
Minor note, you don't need the installation to be editable because presumably you're not modifying the code with your github actions